pkg/flowkit/config/json/emulator_test.go

```diff
@@ -42,3 +43,38 @@ func Test_ConfigEmulatorSimple(t *testing.T) {
 	assert.Equal(t, emulators[0].Name, "default")
 	assert.Equal(t, emulators[0].Port, 9000)
 }
+
+//Emulator config will be empty when first initialized, when parsed from json to config, config should contain default
+//emulator
+func Test_ConfigEmulatorDefault(t *testing.T) {
+	b := []byte(`{
+	 }`)
+
+	var jsonEmulators jsonEmulators
+	err := json.Unmarshal(b, &jsonEmulators)
+	assert.NoError(t, err)
+
+	emulators, err := jsonEmulators.transformToConfig()
+	assert.NoError(t, err)
+
+	assert.Equal(t, emulators[0].Name, config.DefaultEmulatorConfigName)
+	assert.Equal(t, emulators[0].Port, config.DefaultEmulatorPort)
+}
+func Test_ConfigEmulatorBackwardCompatibility(t *testing.T) {
+	b := []byte(`{
+		 "default": {
+				"port": 3569,
+				"serviceAccount": "emulator-account"
+		 }
+	 }`)
+
+	var jsonEmulators jsonEmulators
+	err := json.Unmarshal(b, &jsonEmulators)
+	assert.NoError(t, err)
+
+	emulators, err := jsonEmulators.transformToConfig()
+	assert.NoError(t, err)
+
+	assert.Equal(t, emulators[0].Name, config.DefaultEmulatorConfigName)
+	assert.Equal(t, emulators[0].Port, config.DefaultEmulatorPort)
+}
```

Comment:
Ensures backward compatibility such that we still obtain the default config emulator values when we parse json to config